crates/containerd-shim-wasm/src/sandbox/path.rs

```diff
@@ -19,7 +19,7 @@ pub trait PathResolve {
 }
 
 /// Gets the content of the `PATH` environment variable as an iterator over its components
-pub fn paths() -> impl Iterator<Item = PathBuf> {
+pub(crate) fn paths() -> impl Iterator<Item = PathBuf> {
```

Comment:
this is a breaking change I think? Can you document it in the CHANGELOG?

Reply:
Same as above.